integration_tests/_source_helpers.py

```diff
@@ -0,0 +1,122 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2017 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import contextlib
+import os
+import subprocess
+
+
+@contextlib.contextmanager
+def return_to_cwd():
+    cwd = os.getcwd()
+    try:
+        yield
+    finally:
+        os.chdir(cwd)
+
+
+def call(cmd):
+    subprocess.check_call(cmd, stdout=subprocess.DEVNULL,
+                          stderr=subprocess.DEVNULL)
+
+
+def call_with_output(cmd):
+    return subprocess.check_output(cmd).decode('utf-8').strip()
```

Comment:
nevermind, I see you need them out side of the test case. Still, they should be probably called subprocess utils or something like that.